src/Lexer.php

```diff
@@ -21,7 +21,19 @@
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  *
- * @phpstan-type TToken array{type: string, line: int, value: string|null, deferred: bool, tags?: list<string>, keyword?: string, keyword_type?: string, text?: string, indent?: int, columns?: list<string>}
+ * @phpstan-import-type TLanguage from KeywordsInterface
+ * @phpstan-import-type TKeywordsString from KeywordsInterface
+ * @phpstan-import-type TKeywordsType from KeywordsInterface
+ * @phpstan-import-type TStepKeywordsType from KeywordsInterface
```

Comment:
The StepKeywordType exposed in the `StepNode::getKeywordType` is not related to a string literal belonging to KeywordsInterface. They are a different concept (even if somewhat related).

Note that in `cucumber/messages`, they also have a concept of step keyword type (we don't test equivalence in our CompatibilityTest because our concept is actually a mix between their step keyword type and their pickle step keyword type, because we don't have a separate concept of pickle between the AST and the Behat runner), and they consider `And` and `But` to both be of the `Conjunction` type instead of being separate types once parsed.

Reply:
I didn't touch `StepNode::getKeywordType()`... do you mean it should `@phpstan-return Lexer::TStepKeywordsType`?